tests/integration/pilot/outboundtrafficpolicy/helper.go

```diff
@@ -91,6 +92,92 @@ spec:
     - "{{.ImportNamespace}}/*"
     - "istio-system/*"
 `
+
+	Gateway = `
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-egressgateway
+spec:
+  selector:
+    istio: egressgateway
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - "*.gateway.com"
+  - port:
+      number: 443
+      name: https
+      protocol: TLS
+    hosts:
+    - "*.gateway.com"
+    tls:
+      mode: PASSTHROUGH
+
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: route-via-egressgateway
+spec:
+  hosts:
+    - "*.gateway.com"
```

Comment:
These tests are also canonical examples.
Capturing host as *.gateway.com can be a bit confusing and unusual.
Consider capturing the destination host that the application will use instead.
Like destination.appnamespace .

Reply:
this is for testing traffic to external services, not internal apps. Is there some reason I should be using internal names here for that or did you think this was internal traffic?

Reply:
It’s not internal or external names. The unusual part is that the app code will use foo.gateway.com
We normally expect the app code to use the intended destination name.

Reply:
ah so you mean just explicit hostname instead of wildcard? or do you mean gateway.com is confusing because it isn't the "real" destination like google.com or something?

Reply:
Yes!

Reply:
Ok, I have updated.